nat-lab/tests/uniffi/libtelio_remote.py

```diff
@@ -176,23 +179,35 @@ def probe_pmtu(self, host: str) -> int:
     def get_nat(self, ip: str, port: int) -> libtelio.NatType:
         return self._libtelio.get_nat(ip, port)
 
+    @serialize_error
+    def flush_logs(self):
+        self._logger_cb.logfile.flush()
+
 
 def main():
     object_name = sys.argv[1]
     container_ip = sys.argv[2]
     port = int(sys.argv[3])
 
+    # Cleanup old log files if any exists
+    try:
+        os.remove(TCLI_LOG)
+        os.remove(REMOTE_LOG)
```

Comment:
Is that needed, I thought only `TCLI_LOG` is impacted. Or is this some additional fix/improvement?

Reply:
TBH, this just replicates old behavior:
* https://github.com/NordSecurity/libtelio/blob/c296e61379e9cdab89009119314e082b63209368/nat-lab/tests/uniffi/libtelio_remote.py#L57
* https://github.com/NordSecurity/libtelio/blob/c296e61379e9cdab89009119314e082b63209368/nat-lab/tests/uniffi/libtelio_remote.py#L77